src/loaders/common/loader_common_names.py

```diff
@@ -31,6 +31,10 @@
 # Name for environment argument
```

Comment:
This covers the import files, but are there any other steps in the process that could produce unreadable files in perlmutter without a group change?

Reply:
I think import files might be it. Other files created in the process are not supposed to be manually handled.

Reply:
What if we
* change tool code
* change the parser

... and someone wants to rerun with those changes against unreadable files?

Reply:
I am not getting what you mean. It doesn't sound like a common case. There are a lot of files being created/downloaded. Unless we want to run chown on every files, there will be some unreadable files in some rare cases.

Reply:
So we download the files from workspace A and they're only readable by user B. We find a bug in the gtdb tool container. User C wants to rerun the pipeline but can't because the files aren't readble.

Same thing if the user C just wants to rerun the parser on user B's output

Reply:
yea. Like I mentioned, in order to completely prevent this, we will need to refactor every script that handles files.

Reply:
Ok, so you don't want to do that right now, got it. Ticket instead? At some point this'll cause problems for us, but maybe not for a while